docs/src/pages/components/date-range-picker/CustomDateRangePickerDay.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core';
```

Comment:
Could we use the styled API instead?

```suggestion
```

Reply:
We can do this another time. I haven't looked into styled and have more control with `makeStyles` for now. I'm not looking forward to HOC related type issues with `styled`.

Reply:
WeLet's do this in a follow-up. Marking this review for future reference when I'm being told we "move fast".